### DISCUSSION TITLE
[Enhancement] Please update jwt features

### DISCUSSION THREAD

**Discussion by JaeWangL** [Dotnet-Boxed/Templates#262] [Q&A]
This GraphQL template is awesome and
I am working with my project based on this

Today, Jwt is essential and important
Adding jwt features will be good options

Thanks all very much.

**Comment by RehanSaeed** [comment]
Auth is hard and there are a lot of ways to implement it depending on your OAuth provider. I could provide some code but none of it would work until you plugged in the details of the OAuth provider. So I've left this out. Hope that explains why.
